bin-src/init.ts

```diff
@@ -0,0 +1,134 @@
+import boxen from "boxen";
+import { execaCommand } from 'execa';
+import { writeFile } from 'jsonfile';
+import meow from 'meow';
+import prompts from 'prompts';
+import type { PackageJson } from 'read-pkg-up'
+import readPkgUp from 'read-pkg-up';
+import { getCliCommand, parseNi } from '@antfu/ni';
+
+import noPackageJson from '../node-src/ui/messages/errors/noPackageJson';
+
+export const TestFramework = {
+    STORYBOOK: 'storybook',
+    PLAYWRIGHT: 'playwright',
+    CYPRESS: 'cypress'
+};
+type TestFrameworkType = typeof TestFramework[keyof typeof TestFramework];
+
+export const getPackageManagerInstallCommand = async (args: string[]) => {
+    return await getCliCommand(parseNi, args, { programmatic: true });
+};
+
+export const addChromaticScriptToPackageJson = async ({ packageJson, packagePath }) => {
+    try {
+        const json = {
+            ...packageJson,
+            scripts: {
+                ...packageJson?.scripts,
+                chromatic: `npx chromatic@latest`
+            }
+        }
+        await writeFile(packagePath, json, { spaces: 2 });
+    } catch (e) {
+        console.warn(e)
+    }
+}
+
+export const createChromaticConfigFile = async ({configFile, buildScriptName = null}) => {
+    await writeFile(configFile, {
+        ...(buildScriptName && {
+            buildScriptName
+        })
+    });
+}
+
+export const installArchiveDependencies = async (packageJson: PackageJson, testFramework: TestFrameworkType) => {
+    let installArgs = ['-D', 'chromatic',`chromatic-${testFramework}`, 'storybook@latest', '@storybook/addon-essentials@latest', '@storybook/server-webpack5@latest']
+    const storybookVersion = packageJson?.devDependencies?.storybook || packageJson?.dependencies?.storybook
+    const essentialsVersion = packageJson?.devDependencies?.['@storybook/addon-essentials'] || packageJson?.dependencies?.['@storybook/addon-essentials']
+    if(storybookVersion && essentialsVersion) {
```

Comment:
It would be possible for somebody to have `storybook` installed, but not `addon-essentials`, right? If that's the case, this will upgrade their `storybook` version rather than installing `addon-essentials` at the current sb version.